src/Module.php

```diff
@@ -40,6 +40,17 @@ public function getViewHelperConfig()
                     /** @var Rollout $rollout */
                     $rollout = $serviceLocator->get('zf2_rollout');
                     return new View\Helper\IsActive($rollout);
+                },
+                'rollout_description' => function (AbstractPluginManager $pluginManager) {
+                    $serviceLocator = $pluginManager->getServiceLocator();
+                    $rolloutConfig = $serviceLocator->get('zf2_rollout_config');
+
+                    $features = [];
+                    if (isset($rolloutConfig['features'])) {
```

Comment:
Not sure the if is necessary ... since the 'features' key is included in the config/module.inc.php file

Reply:
not necessarily since the _features_ key  in the config is not required